akka-persistence/src/main/scala/akka/persistence/SnapshotProtocol.scala

```diff
@@ -24,6 +24,7 @@ final case class SnapshotMetadata(persistenceId: String, sequenceNr: Long, times
  */
 @SerialVersionUID(1L)
 final case class SaveSnapshotSuccess(metadata: SnapshotMetadata)
+  extends SnapshotProtocol.Response // Internal snapshot acknowledgement.
```

Comment:
1) I'd still remove the comment. If something is internally and user used, users may look for it, we shouldn't confuse them with the internal comment (used in akka when we mean "do not touch!"). The trait is OK ofc.
2) sounds good.
3) right, good then.

Summing up I'd only remove the comments